docs/get_start.rst

```diff
@@ -9,29 +9,34 @@ Quick start of infrasim-compute application
 Command interfaces
 ~~~~~~~~~~~~~~~~~~~~~
 
-#. Initialization (you need do it once) ::
+Initialize infrasim. (you need to do it once)::
 
     sudo infrasim init
 
+Start services.
+
 #. Start Infrasim Service::
 
     sudo infrasim node start
 
    Verify your service by `VNC and IPMI <startInterface_>`_
 
-#. Status and version number check::
+#. Start IPMI Console::
+
+    sudo ipmi-console start
+
+Status and version number check.
 
     sudo infrasim node status
+    
     sudo infrasim version
 
+Stop services.
```

Comment:
Need to be highlight, adding "#." as prefix or something.

Or adding more explanation like "Shutdown virtual server"